kong/clustering/control_plane.lua

```diff
@@ -77,6 +77,17 @@ local function is_timeout(err)
 end
 
 
+local function extract_dp_cert(cert)
+  local expiry_timestamp = cert:get_not_after()
+  -- values in cert_details must be strings
+  local cert_details = {
+    expiry_timestamp = expiry_timestamp,
```

Comment:
what if `cert:get_not_after()` returns `nil` or `false`?
```suggestion
    expiry_timestamp = expiry_timestamp or "unknown",
```

Reply:
update all, many thanks